src/FileIo.cpp

```diff
@@ -103,18 +103,20 @@ QString FileIo::pathFromUrl(QString const& _url)
 	return path;
 }
 
-void FileIo::makeDir(QString const& _url)
+bool FileIo::makeDir(QString const& _url)
```

Comment:
If you change the semantics of this function, you should also change how it is used.
In most places where this is used, the assumption is that the directory can be used afterwards and is clear.